text2graph/gkm/features/stratigraphy.py

```diff
@@ -0,0 +1,266 @@
+from enum import IntEnum
+from dataclasses import dataclass
+from rdflib import Graph, Literal, RDF, RDFS, Namespace, URIRef, BNode
+
+from text2graph.macrostrat import get_all_intervals
+from text2graph.schema import Stratigraphy, RelationshipTriplet
+from text2graph.gkm.namespace import GSOC, GSOG, GSGU, GSPR, GST, MSL
+from text2graph.gkm.features.general import (
+    entity_name,
+    add_macrostrat_query_and_entity,
+)
+
+
+def object_node_stratigraphy(triplet_object: Stratigraphy) -> URIRef:
+    """
+    create strat_name node/URIRef for object/strat_name dict
+    :param triplet_object: subject/strat_name dict
+    :return: URIRef
+    """
+    try:
+        object_name = triplet_object.strat_name_long
```

Comment:
This line assumes triplet is already hydrated and is an known entity. There are a few cases it will not work.

1) If a user call the API with option: hydrate=False.
2) Unknown entity (LLM suggest it is a Stratname, but it cannot be found in Macrostrat)

How we should deal with the PROV in these senarios?

Reply:
test case: 

```python
from text2graph.llm import fast_llm_graph_from_search
y = await fast_llm_graph_from_search("iron mines in Minnesota", top_k=3, ttl=False)
```

This will return an empty str.

Reply:
Maybe another example, in case you don't have the cached data on your machine:

```python
from text2graph.alignment import get_alignment_handler
from text2graph.llm import llm_graph_from_search
from text2graph.prompt import get_prompt_handler

y = await llm_graph_from_search(
    "iron mines in Minnesota",
    top_k=3,
    ttl=True,
    model="mixtral",
    alignment_handler=get_alignment_handler("strat_name"),
    prompt_handler=get_prompt_handler("stratname_v3"),
)
```

Reply:
@JasonLo Good points. I've changed it to defaulting to the triplet.object.strat_name, and only replacing that with strat_name_long if its truthy (i.e. not "") and != "None".